app/views/repositories/annotate.html.erb

```diff
@@ -42,7 +42,7 @@ See doc/COPYRIGHT.rdoc for more details.
             <%= (revision.identifier ? link_to_revision(revision, @project) : format_revision(revision)) if revision %></td>
           <td class="author"><%= h(revision.author.to_s.split('<').first) if revision %></td>
           <td class="line-code">
-            <pre><%= line %></pre>
+            <pre><%= raw line.html_safe %></pre>
```

Comment:
Ah, thanks. Missed this one while fixing the underlying issue you mentioned. I agree and will move this into a separated PR.